gui/src/app/pages/HomePage/saveAsGitHubGist.ts

```diff
@@ -0,0 +1,74 @@
+import { Octokit } from "@octokit/core";
+
+const saveAsGitHubGist = async (
+  files: { [key: string]: string },
+  o: { defaultDescription: string; personalAccessToken: string },
+) => {
+  const { defaultDescription, personalAccessToken } = o;
+  const description = prompt(
```

Comment:
Oh also -- this prompt is redundant with the title box already on the save page, right?

I guess it's nice as an "are you sure"?

Reply:
This was intentional. When saving to a gist it's important to specify an appropriate title/description, and I often find myself forgetting to do this, even when the option is there in the table, and so this is a very helpful opportunity.